src/Uno.UI.Tests/Windows_UI_XAML_Controls/StackPanelTest/Given_StackPanel.cs

```diff
@@ -21,443 +15,425 @@ public class Given_StackPanel : Context
 		[TestMethod]
 		public void When_Vertical_And_SimpleLayout()
 		{
-			var SUT = new StackPanel() { Name = "test", Orientation = Orientation.Vertical };
+			var SUT = new StackPanel { Name = "test", Orientation = Orientation.Vertical };
 
 			var c1 = SUT.AddChild(
-				new View { Name = "Child01", RequestedDesiredSize = new Windows.Foundation.Size(10, 8) }
+				child: new View { Name = "Child01", RequestedDesiredSize = new Size(width: 10, height: 8) }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'Child01' 14 times.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826507)